rpcs/ledgerService.go

```diff
@@ -60,19 +60,25 @@ type LedgerForService interface {
 	GetCatchpointStream(round basics.Round) (ledger.ReadCloseSizer, error)
 }
 
+// httpGossipNode is a reduced interface for the gossipNode that only includes the methods needed by the LedgerService
+type httpGossipNode interface {
```

Comment:
Should be able to reduce a mock somewhere based on this right

Reply:
that's the reason why I added this type - to mock only two methods instead of 15.